### PR TITLE
Good Housekeeping, Issue #1

### PR DESCRIPTION
**Pull request by henricook**
I set out to clear some of the Fasttrack/old Faststream object Detritus, because i'm about to work on some reports and a lot of it would have been really confusing.

I ended up refactoring model.Commands - the file filled with loads of classes, any that were still being used were moved to their own files (and corresponding imports and tests adjusted)

I've also removed any old assessment centre code that caught my eye (ApplicationAssessment and some old case classes mainly)

This is all just move and shift, should be a skim - don't let the 107 files scare you! :1st_place_medal:

### CODE REVIEW

File: app/model/persisted/eventschedules/SkillType.scala [hunk 1]
```diff
@@ -21,8 +21,6 @@ import reactivemongo.bson.{ BSON, BSONHandler, BSONString }
 
 import scala.language.implicitConversions
 
-
-@deprecated("We should get rid of this enum and change code accordingly to use AssessorSkill (renamed to SkillType)", since = "10/07/2017")
```
Comment (henricook) [inline app/model/persisted/eventschedules/SkillType.scala]: AssessorSkill depends on SkillType - discussed with the author of the message and decided to remove

**Comment by fayimora** [comment]
@henricook LGTM! 👍 ⚡️

**Comment by fayimora** [comment]
@henricook Want a merge on this now?

**Comment by henricook** [comment]
Yes please

**Pull request merged**